osudroidresource/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -1,62 +1,157 @@
+	<string name="replay_cantdownload">Não foi possivel baixar o replay</string>
+	<string name="replay_wrongmap">O replay não combina com o beatmap</string>
+	<string name="replay_corrupted">O arquivo de replay esta corrompido</string>
+
+	<string name="online_registration">Registração</string>
+	<string name="online_registration_process">Registrando...</string>
+	<string name="online_loadrecord">Carregando detalhes gravados...</string>
+	<string name="online_invlogin">Login inválido</string>
+	<string name="online_invpassword">A senha tem que ser de 4-32 carácteres</string>
+	<string name="online_invconfirm">As senhas não conbinam</string>
+	<string name="online_invemail">Invaild e-mail</string>
+	<string name="online_regcomplete">Registração completa. reinicie o app para fazer login</string>
+
+	<!-- Color Picker -->
+	<string name="dialog_color_picker">Seletor de cor</string>
+	<string name="press_color_to_apply">Pressione na cor para aplicar</string>
+
+	<string name="network_not_connected">Conexão com a internet falhou, por favor cheque as configurações de internet</string>
+	<string name="xml_parser_failed">Excessão na análise de dados</string>
+	<string name="io_exception_error">Excessão no fluxo de arquivos</string>
+	<string name="http_status_code_error">Excessão de rede, código do erro：%d</string>
+	<string name="http_exception_error">Excessões de rede, timeout solicitado</string>
+	<string name="socket_exception_error">Excessão de rede, tempo limite atingido</string>
+
+	<string name="binfoStr1" formatted="false">Duração: %s BPM: %s Combo: %d</string>
+	<string name="binfoStr2" formatted="false">Círculos: %d Sliders: %d Spinners: %d (MapId: %d)</string>
+
+	<!-- Crash -->
+	<string name="crash">Desculpa, situação anormal, fechando o jogo.</string>
+	<string name="app_run_code_error">Erro fatal encontrado</string>
+	<string name="osu_parser_error">Dificuldades tentando encontrar o erro</string>
+	<string name="require_storage_permission">O jogo não consegue rodar sem permissões de armazenamento. favor habilitar a permissão manualmente</string>
+
+	<string name="update_dialog_title">Nova versão disponivel</string>
+	<string name="update_dialog_button_confirm">Atualizar agora</string>
+	<string name="update_dialog_button_later">Depois</string>
+	<string name="update_info_checking">Procurando por atualizações, favor ser paciente</string>
+	<string name="update_info_runs_latest">Você ja esta atualizado :D</string>
+
+	<string name="test_ed_storyboard">Ver storyboards (experimental)</string>
+
+	<string name="mod_precise_is_unrank_now">O mod PRECISE não esta ranqueado por enquanto</string>
+
+
+	<!-- new data after 1.6.7(beta1) -->
+    <string name="beta_checking_upgrade">Procurando por atualizações...</string>
+	<string name="beta_check_upgrade_error">Falha ao procurar atualizações, tente novamente depois</string>
+	<string name="beta_you_are_the_latest_version">Você ja esta usando a ultima versão disponível</string>
```

Comment:
``Você já está utilizando a última versão disponível``

Reply:
I think this could be better with ``Você já está utilizando a versão mais recente disponível`` @EdrowsLuo

Reply:
fixed

Reply:
remove ``disponivel``, I'm dumb sometimes xd

Reply:
i did it :p